coercions/packagesFromJsonFileCoercion.js

```diff
@@ -0,0 +1,27 @@
+'use strict';
+let fs = require('fs'),
+    packages = require('./packagesCoercion');
+
+module.exports = function (jsonFilePath) {
+    const fileContent = JSON.parse(fs.readFileSync(jsonFilePath, "utf8"));
+    const dependencies = attachNameAndVersion(fileContent.dependencies);
+    const devDependencies = attachNameAndVersion(fileContent.devDependencies);
```

Comment:
dev-dependencies flag will dowload all the dev dependencies of each dependency from the json file if set to true.
would u rather to set another flag to dowload dev dependencies from the json file?